Encounter Me/Pages/Registration.razor

```diff
@@ -27,6 +34,44 @@
+                            <InputRadioGroup Name="Faction" @bind-Value="registrationUserModel.Faction" class="form-check-inline" id="Faction">
+                                <label class="btn btn-outline-warning">
+                                    <InputRadio Name="Faction" class="inputRadio" Value="Factions.Yellow" autocomplete="off" /> Yellow
+                                </label>
+                                <label class="btn btn-outline-primary">
+                                    <InputRadio Name="Faction" class="inputRadio" Value="Factions.Blue" autocomplete="off" />Blue <br>
+                                </label>
+                                <label class="btn btn-outline-danger">
+                                    <InputRadio Name="Faction" class="inputRadio" Value="Factions.Red" autocomplete="off" />Red <br>
+                                </label>
+                                <label class="btn btn-outline-success">
+                                    <InputRadio Name="Faction" class="inputRadio" Value="Factions.Green" autocomplete="off" />Green <br>
+                                </label>
+                            </InputRadioGroup>
+                        </div>
+                        <ValidationMessage For="@(() => registrationUserModel.Faction)" />
+                    </div>
+
+                    @*<div class="col-md-12 mb-3">
+                            <div class="btn-group btn-group-toggle" data-toggle="buttons">
+                                <label class="btn btn-outline-warning">
+                                    <input type="radio" class="form-check-input" name="fraction" @bind="@registrationUserModel.Fraction" @onclick="@((e) => { registrationUserModel.Fraction = Fractions.Yellow;})" id="option1" required=""> Yellow
+                                </label>
+                                <label class="btn btn-outline-primary">
+                                    <input type="radio" class="form-check-input" name="fraction" @bind="@registrationUserModel.Fraction" @onclick="@((e) => { registrationUserModel.Fraction = Fractions.Blue;})" id="option2" required=""> Blue
+                                </label>
+                                <label class="btn btn-outline-danger">
+                                    <input type="radio" class="form-check-input" name="fraction" @bind="@registrationUserModel.Fraction" @onclick="@((e) => { registrationUserModel.Fraction=Fractions.Red;})" id="option3" required=""> Red
+                                </label>
+                                <label class="btn btn-outline-success">
+                                    <input type="radio" class="form-check-input" name="fraction" @bind="@registrationUserModel.Fraction" @onclick="@((e) => {registrationUserModel.Fraction=Fractions.Green;})" id="option4" required=""> Green
+                                </label>
+                            </div>
+                        </div>*@
```

Comment:
Is this a different a way to do faction selection? Would be nice to have some comment about this commented part.

Reply:
As I understand, new table is created for table ? I think thats a good idea, because now some additional logic with factions gamification.

Reply:
It basically looks the same - the main difference was error message - in the commented part error message is a default Cromium message which cannot be styled. I left it as I forgot to delete it...

Reply:
I'll be doing a clean up next - will delete old comments and refactor the structure of the project - move classes to folders et. ct